### PR TITLE
[ENH] add schema in appendix

### LINKED ISSUE #1541: mention the schema in the specification
Unless I am mistaken we are not mentioning the existence of the schema in the spec itself.

This should probably be done, mention where to find the schema and / or link to the schema documentation.

Note that it is "possible" to access the readme of the schema doc via the spec rendering:

https://bids-specification.readthedocs.io/en/latest/schema/index.html

But the rendering is kinda of broken and we should maybe not render this page and instead have an appendix for the schema, no?

### PR DESCRIPTION
**Pull request by Remi-Gau**
closes #1541

### CODE REVIEW

File: src/appendices/schema.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# BIDS schema
+
+The BIDS specification exist as a schema.
+The BIDS schema is a machine readable representation of the BIDS Standard written in .
+It is (by and large) the BIDS Specification, but written in a declarative form.
+
+The BIDS schema is available in two machine readable formats:
+
+- as a set of [YAML](https://en.wikipedia.org/wiki/YAML) files in the [BIDS specifiation repository](https://github.com/bids-standard/bids-specification/src/schema)
```
Comment (nellh) [inline src/appendices/schema.md:9]:
```suggestion
- as a set of [YAML](https://en.wikipedia.org/wiki/YAML) files in the [BIDS specification repository](https://github.com/bids-standard/bids-specification/src/schema)
```

File: src/appendices/schema.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+- as a [single json file](https://bids-specification.readthedocs.io/en/stable/schema.json)
```
Comment (nellh) [inline src/appendices/schema.md:10]:
```suggestion
- as a [single dereferenced json file](https://bids-specification.readthedocs.io/en/stable/schema.json)
```
I think specifying the dereferenced difference is useful to note here.

File: src/appendices/schema.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
+# BIDS schema
+
+The BIDS specification exist as a schema.
+The BIDS schema is a machine readable representation of the BIDS Standard written in .
+It is (by and large) the BIDS Specification, but written in a declarative form.
```
Comment (effigies) [inline src/appendices/schema.md:5]:
```suggestion
The BIDS schema is a machine readable representation of the BIDS standard,
written in a custom YAML format.
The goal of the schema is to provide a single source for rendering the specification
and validating BIDS datasets, reducing the scope for inconsistencies.
Third party tools may also use the schema to write code that will adapt to additions
to the BIDS standard.
```

File: src/appendices/schema.md [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+The BIDS schema is available in two machine readable formats:
+
+- as a set of [YAML](https://en.wikipedia.org/wiki/YAML) files in the [BIDS specifiation repository](https://github.com/bids-standard/bids-specification/src/schema)
+- as a [single json file](https://bids-specification.readthedocs.io/en/stable/schema.json)
+
+A light-weight introduction to the schema can be found [here](https://bids-extensions.readthedocs.io/en/latest/schema/).
+
+A full description of the schema can be found on this [website](https://bidsschematools.readthedocs.io/en/latest/?badge=latest)
+where you will also find the documentation for the python package
+to interact with the schema, [bidsschematools](https://pypi.org/project/bidsschematools/).
```
Comment (effigies) [inline src/appendices/schema.md:16]:
```suggestion
A didactic walkthrough of the schema can be found in the [BEP Guide](https://bids-extensions.readthedocs.io/en/latest/schema/),
and a complete description is available in the [`bidsschematools` documentation](https://bidsschematools.readthedocs.io/en/latest/).
[`bidsschematools`](https://pypi.org/project/bidsschematools/) is a Python package sourced
from the specification repository, and includes the necessary code to render the specification
and filename validation.
```

**Review by effigies — COMMENTED** [review]
Some thoughts on wording. Feel free to reject, but note:

* L4 ended with a dangling preposition
* "here" and "this website" are [bad anchor text](https://developers.google.com/search/docs/crawling-indexing/links-crawlable#write-good-anchor-text).

**Comment by Remi-Gau** [comment]
thanks for the feedback @nellh @effigies

**Review by sappelhoff — APPROVED** [review]

**Review by nellh — APPROVED** [review]

**Pull request merged**